src/quo2/components/selectors/selectors.cljs

```diff
@@ -0,0 +1,125 @@
+(ns quo2.components.selectors.selectors
+  (:require [reagent.core :as reagent]
+            [quo.react :as react]
+            [quo.react-native :as rn]
+            [quo2.foundations.colors :as colors]
+            [cljs-bean.core :as bean]
+            [quo2.components.icon :as icons]))
+
+(defn get-color [value disabled?]
```

Comment:
I think `checked?` might be a better name for the first param

Reply:
good call :)